src/settings.ts

```diff
@@ -373,32 +279,97 @@ export class xAxisLabelsSettings extends BaseFontCardSettings {
         value : {value: "#000"}
     });
 
+    lineLength = new formattingSettings.Slider({
+        name: "lineLength",
+        displayNameKey: "Visual_LineLength",
+        value: 100,
+        options: {
+            minValue: {
+                type: powerbi.visuals.ValidatorType.Min,
+                value: xAxisLabelsSettings.MinLineLength
+            },
+            maxValue: {
+                type: powerbi.visuals.ValidatorType.Max,
+                value: xAxisLabelsSettings.MaxLineLength
+            }
+        }
+    })
+
     name: string = "xAxisLabelsGroup";
     displayName?: string = "X-Axis labels";
-    slices: FormattingSettingsSlice[] = [this.color, this.font];
+    displayNameKey?: string = "Visual_XAxisLabels";
+    slices: FormattingSettingsSlice[] = [this.color, this.font, this.lineLength];
 }
 
-export class yAxisLabelsSettings extends FormattingSettingsSimpleCard {
+export class yAxisLabelsSettings extends BaseFontCardSettings {
     showOverlapping = new formattingSettings.ToggleSwitch({
-        name: "showOverlaping",
+        name: "showOverlapping",
         displayName: "Show overlapping labels",
+        displayNameKey: "Visual_Show_Labels_Overlapping",
+        description: "Show labels even if they overlap",
+        descriptionKey: "Visual_Description_Labels_Overlapping",
         value: true
     });
 
     show = new formattingSettings.ToggleSwitch({
-        name: "showY",
+        name: "y_show",
         displayName: "Show Y-Axis labels",
         value: true
     });
+
+    showCustomColor = new formattingSettings.ToggleSwitch({
+        name: "showYLabelCustomColor",
+        displayNameKey: "Visual_Show_Custom_Color",
```

Comment:
Let's change it to `Visual_Use_Custom_Color`, also replacing display name to "use" instead of "show".